src/components/elements/table/members-table.tsx

```diff
@@ -65,10 +76,13 @@ const MembersTable: React.FC<Props> = ({ rows, isLoading }) => {
                         {item["email"]}
                       </div>
                     </td>
+                    <td className="w-1/4 space-x-2 px-5 py-4 text-right align-middle">
```

Comment:
Rather than adding an another cell to the table, I think this type of contextual information makes more sense to add to the name + email cell. That saves having to add an additional cell to the Admin row and also having this float in the middle of the table. Probably lay it out like this

```
[LOCK title="disabled"] [Name] 
[Email]
```

This removes the "Disabled" text and just uses the red lock. Then put "disabled" as title text on the element. So available during hover, but doesn't add additional spacing. Keeping the red color is great though because it visually makes for an easy scan. 

Do you think having a bit of context in the title as well makes sense? For instance: "User disabled. Disabled users cannot do x, y, z..."

Reply:
Sounds good, I think I would switch the name and lock-icon, so that visually the names in all rows are aligned since the disabled status is secondary information. What do you think?

I am happy to use the title "User disabled. Disabled users cannot log in.".

Reply:
I like that solution 🙌